discovery-provider/plugins/notifications/src/processNotifications/mappers/announcement.ts

```diff
@@ -85,10 +93,23 @@ export class Announcement extends BaseNotification<AnnouncementNotificationRow>
         )
         await this.incrementBadgeCount(userId)
       }
+
       if (
+        isLiveEmailEnabled &&
         userNotificationSettings.shouldSendEmail({ receiverUserId: userId })
       ) {
-        // TODO: Send out email
+        const notification: AppEmailNotification = {
+          receiver_user_id: userId,
+          ...this.notification
+        }
+        await sendNotificationEmail({
+          userId: userId,
+          email: userNotificationSettings.email?.[userId].email,
+          frequency: 'live',
```

Comment:
we gotta check if the user has selected 'live' notifications as their preference in `shouldSendEmail`, bc users could have chosen daily/weekly as their preference, in which case they should not receive this

Reply:
oh right added